packages/astro/src/compiler/codegen/index.ts

```diff
@@ -768,7 +768,7 @@ async function compileHtml(enterNode: TemplateNode, state: CodegenState, compile
             }
             if (parent.name === 'code') {
               // Special case, escaped { characters from markdown content
-              text = node.raw.replace(/&#x26;#123;/g, '{');
+              text = node.raw.replace(/LEFT_CURLY_BRACKET_FIX\0/g, '{');
```

Comment:
Can we give this an astro-specific name so that anyone debugging knows where it's coming from? How about: `ASTRO_ESCAPED_LEFT_CURLY_BRACKET` ?

Reply:
Yes. Done in c094059.